relay/update_client.go

```diff
@@ -0,0 +1,59 @@
+package relay
+
+import (
+	"context"
+	"fmt"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+	"github.com/datachainlab/lcp-go/relay/elc"
+)
+
+func updateClient(ctx context.Context, chunkSize uint32, client LCPServiceClient, anyHeader *types.Any, elcClientID string, includeState bool, signer []byte) (*elc.MsgUpdateClientResponse, error) {
```

Comment:
Please provide more context for each error returned. (e.g., `return nil, fmt.Errorf("failed to UpdateClientStream: %w", err)`)

Reply:
Fixed by https://github.com/datachainlab/lcp-go/commit/d8262d6dda4a5416cc52120973f30f55257db41c